parcel-resolver-veda/defaults.js

```diff
@@ -4,7 +4,8 @@ const defaultStrings = {
   stories: {
     one: 'Story',
     other: 'Stories'
-  }
+  },
+  storiesBanner: 'Explore the guided narratives below to discover how NASA satellites and other Earth observing resources reveal a changing planet.'
```

Comment:
❓ : Below you do `getString('storiesBanner').other` but the default has no `other` key? Would this be alright?

Also is here a config template? Possibly as documentation and as well as usage for a possible validator that I talked about below..

Reply:
@sandrahoang686 For any string that gets added, there should be a default in the `parcel-resolver-veda/defaults.js`. This will ensure that the optional chaining is not need.

Then, to simplify configuration, the user can define either an object with `one` and `other` or a single string. The veda module will take care of normalizing the format, ensuring that there is always `one` and `other` for the implementer to use, with both values being the same if the user just provided a string. In this way the function return is always the same.
There was a bug in the code that performed this operation, but you can see it fixed here: https://github.com/NASA-IMPACT/veda-ui/blob/da4527b6d59110576b13ae12fdc9f15b97ad39f5/parcel-resolver-veda/defaults.js